src/content/templates/pages/cloud_storage.md

```diff
@@ -8,11 +8,6 @@ hidden: true
 
 You can fetch your 3rd-party object storage assets with a Cloudflare Worker so that they can be cached in the Cloudflare CDN.
 
-- [Overview](#overview)
```

Comment:
note this is the only template page with a TOC right now. We might implement a TOC on other pages later, but for now this is not necessary